src/app/researcher/study/request/[memberIdentifier]/study-proposal-form-steps.tsx

```diff
@@ -23,6 +24,13 @@ export const StudyProposalFormSteps: React.FC<{ memberId: string }> = ({ memberI
             return await onCreateStudyAction({ memberId, studyInfo: formValues })
         },
         onSuccess() {
+            notifications.show({
+                title: 'Study Proposal Submitted',
+                message:
+                    'Your proposal has been successfully submitted to the reviewing organization. Check your dashboard for status updates.',
+                color: 'green',
+                position: 'top-right',
```

Comment:
by omitting the `position` property it will default to whatever mantine has set, which right now is bottom-right. i confirmed with iris that we want to use top-right for all the notifications, so would you be able to remove this line here and then add the following to where we have <Notifications> defined?
```
import { Notifications } from '@mantine/notifications';

function Demo() {
  return <Notifications position="top-right"  />;
}
```